crates/turborepo-lib/src/commands/prune.rs

```diff
@@ -52,6 +52,32 @@ lazy_static! {
             RelativeUnixPath::new(".npmrc").unwrap(),
             Some(CopyDestination::Docker)
         ),
+        (
+            RelativeUnixPath::new(".yarnrc.yml").unwrap(),
+            Some(CopyDestination::Docker)
+        ),
+    ];
+    static ref ADDITIONAL_DIRECTORIES: Vec<(&'static RelativeUnixPath, Option<CopyDestination>)> = vec![
+        (
+            RelativeUnixPath::new(".yarn/patches").unwrap(),
+            Some(CopyDestination::Docker)
+        ),
+        (
+            RelativeUnixPath::new(".yarn/plugins").unwrap(),
+            Some(CopyDestination::Docker)
+        ),
+        (
+            RelativeUnixPath::new(".yarn/releases").unwrap(),
+            Some(CopyDestination::Docker)
+        ),
+        (
+            RelativeUnixPath::new(".yarn/sdks").unwrap(),
```

Comment:
Reading the description of what is in `.yarn/sdks` I'm not sure if most Docker images want these files.